tcf_website/templates/reviews/review_stats.html

```diff
@@ -0,0 +1,16 @@
+<div class="large-stats-container container-lg">
+    <div class="d-flex">
+        <div class="col-xs-4 large-stats">
+            <h3>Total Reviews Written</h3>

```

Comment:
For verbosity reasons might want to remove the redundant "Review" word from each stat panel, since you already have the "My Reviews" header right above that

Reply:
Yeah, I also thought it was kinda redundant but used the original wording for parity.